### ISSUE TITLE
Missing Binlog Stats

### ISSUE THREAD

**Issue by gordan-bobic** [shatteredsilicon/ssm-submodules#233]
At least on MySQL 8.0.x, with st-sideload-relay, it looks like the master's binlog generation isn't captured on the MySQL Replication dashboard.

Relay Log Space on the slaves shows 16 Exabytes
Relay Log Written Hourly shows 11 ZettaBytes
Obviously these are not true.

I am not sure if this is the result of using st-sideload-relay - it likely is.
Is there a way we can work around this and make it record accurate data regardless of whether st-sideload-relay is used?

**Comment by oblitorum** [comment]
Yeah, that's the side effect of using st-sideload-relay currently.
Those 2 dashboards use the `Relay_Log_Space` value from `SHOW SLAVE STATUS` output. I think one way to get around this is to sum up the size of actual relay log files, for example, sum up the size number from this output:

```bash
root@17fabd512373:/# stat -c "%s %n" /var/lib/mysql/mysqld-relay-bin.*
310 /var/lib/mysql/mysqld-relay-bin.000109
2291146 /var/lib/mysql/mysqld-relay-bin.000110
310 /var/lib/mysql/mysqld-relay-bin.000111
2257599 /var/lib/mysql/mysqld-relay-bin.000112
310 /var/lib/mysql/mysqld-relay-bin.000113
```

I got this output after I used `st-sideload-relay` and the `Relay_Log_Sapce` went crazy, so the relay log files are not affected I think.
BTW, that `Relay_Log_Space` value from `SHOW SLAVE STATUS` output seems to be **in-memory** value, I restarted the mariadb server and the value back to normal, I think it re-calculated the size from all present relay log files.

**Comment by gordan-bobic** [comment]
I see. I'm going to bump this from .4 to .5 release since it only affects systems where st-sideload-relay is in use.
Then we can discuss what the best course of action is.

**Comment by oblitorum** [comment]
OK, got it
